docs/reference/compatibility.txt

```diff
@@ -537,19 +537,29 @@ Mongoid is used as a drop-in replacement.
      - |checkmark| [#rails-activejob-dependency]_
 
    * - ``ActiveModel``
-     - |checkmark|
+     - |checkmark| [#rails-activemodel-dependency]_
 
    * - ``ActiveStorage``
      - |x| [#rails-activerecord-dependency]_
 
+   * - ``ActiveSupport``
+     - |checkmark| [#rails-activesupport-dependency]_
+
 .. [#rails-actioncable-dependency] There is currently no MongoDB adapter for
    ``ActionCable``, however any existing adapter (such as `Redis <https://guides.rubyonrails.org/action_cable_overview.html#redis-adapter>`_)
    can be used successfully in conjunction with Mongoid models
 
 .. [#rails-activerecord-dependency] Depends directly on ``ActiveRecord``
 
-.. [#rails-activejob-dependency] Serialization of BSON & Mongoid objects  works best
-   if you explicitly send ``ObjectId``'s as strings, and reconstitute them in the job:
+.. [#rails-activemodel-dependency] ``Mongoid::Document`` includes ``ActiveModel::Model``
+   and leverages ``ActiveModel::Validations`` for validations
+
+.. [#rails-activesupport-dependency] ``Mongoid::Document`` extends ``ActiveSupport::Concern``. Mongoid
```

Comment:
Would suggest to remove "`Mongoid::Document`` extends ``ActiveSupport::Concern``" here. I think this can just be:

> Mongoid requires ``ActiveSupport`` and uses it extensively, including ``ActiveSupport::TimeWithZone`` for time handling.